entries/f/flyingblue.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Flying Blue": {
+    "domain": "flyingblue.com",
+    "additional-domains": [
+      "airfrance.com",
+      "klm.com"
+    ],
```

```suggestion
    "additional-domains": [
      "airfrance.com",
      "klm.com",
      "flyingblue.am",
      "flyingblue.at",
      "flyingblue.be",
      "flyingblue.bz",
      "flyingblue.ca",
      "flyingblue.cn",
      "flyingblue.co.in",
      "flyingblue.co.nz",
      "flyingblue.co.uk",
      "flyingblue.co.za",
      "flyingblue.com.au",
      "flyingblue.com.co",
      "flyingblue.com.mx",
      "flyingblue.com.ph",
      "flyingblue.com.tw",
      "flyingblue.com.ve",
      "flyingblue.de",
      "flyingblue.es",
      "flyingblue.fr",
      "flyingblue.la",
      "flyingblue.nl",
      "flyingblue.pl",
      "flyingblue.us"
    ],
```

Flying Blue is also available on country-specific domains